Configuration.props

```diff
@@ -19,7 +19,8 @@
     <AndroidMxeInstallPrefix Condition=" '$(AndroidMxeInstallPrefix)' == '' ">$(AndroidToolchainDirectory)\mxe</AndroidMxeInstallPrefix>
     <AndroidSdkDirectory>$(AndroidToolchainDirectory)\sdk</AndroidSdkDirectory>
     <AndroidNdkDirectory>$(AndroidToolchainDirectory)\ndk</AndroidNdkDirectory>
-    <AndroidSupportedAbis Condition=" '$(AndroidSupportedAbis)' == '' ">host-$(HostOS):armeabi-v7a</AndroidSupportedAbis>
+    <AndroidSupportedDeviceAbis>armeabi-v7a</AndroidSupportedDeviceAbis>
```

Comment:
Not really - we have Configuration.Override.props.in which explicitly declares AndroidSupportedAbis. If you want everyone to change the property, you should also make changes to that file.